source/incident-collaboration/launching-playbooks.rst

```diff
@@ -1,45 +1,43 @@
-- ``/incident commander [@username]`` - Show or change the current commander.
-- ``/incident info`` - Show a summary of the current incident.
+- ``/playbook run`` - Start a playbook run.
+- ``/playbook end`` - End a playbook run.
+- ``/playbook update`` - Update the current playbook run's status.
+- ``/playbook restart`` - Restart an ended playbook run.
+- ``/playbook check [checklist item]`` - Check/uncheck the specified task.
+- ``/playbook announce ~[channels]`` - Announce the current playbook run in other channels.
+- ``/playbook list`` - List all your running playbooks.
+- ``/playbook owner [@username]`` - Show or change the current playbook run owner.
+- ``/playbook info`` - Show a summary of the current playbook's activities.
 
-To run a playbook use the ``/incident start`` slash command from any channel. You can also use the desktop or browser to:
+To run a playbook use the ``/playbook start`` slash command from any channel. You can also use the desktop or browser to:
 
- * Select the **Incidents** icon in the channel header, and select **Start Incident** from the right-hand sidebar.
- * Use the context menu of a post and select **Start incident**.
+ * Select the **Playbook** icon in the channel header, and select **Run playbook** from the right-hand sidebar.
+ * Use the context menu of a post and select **Run playbook**.
 
-Running a playbook opens an interactive dialog. Provide an incident name and select a playbook to be used with the incident. Optionally provide a description to offer immediate context into the newly started incident. Only playbooks of which you are a member are listed in the **Playbook** drop-down menu.
+Running a playbook opens an interactive dialog. You can create a new playbook or select an existing playbook. Provide a name for your playbook run, then select **Start run**. Only playbooks you belong to are listed in the **Playbook** drop-down menu. The creator of a playbook run is automatically added as the first member and becomes the run owner.
 
-The creator of a playbook run is automatically added as the first member and becomes the commander.
+When the playbook run is created, it's announced in the configured channel. If that channel is archived or deleted, the Playbook bot posts a notification in the incident channel.
 
-When the playbook run is created, it's announced in the configured channel. If that channel is archived or deleted, the bot posts a notification in the incident channel.
-
-Joining an incident
-~~~~~~~~~~~~~~~~~~~
+Joining a playbook run
+~~~~~~~~~~~~~~~~~~~~~~
 
-Join an incident by joining the incident channel. If the incident channel is public, you may join the channel without permission. Search for and join the channel via **Browse Channels**.
+Select the **Playbook** icon in the top menu bar to see a list of runs in progress. If the channel is public, you may join the channel without permission. You can also search for and join the channel via **Browse Channels**.
```

Comment:
Unfortunately, the behaviour here is to only list the runs to which you already have access. To join a run in progress, we should send the user to the backstage runs list.